.github/workflows/reusable-codeception-tests-public.yaml

```diff
@@ -84,6 +84,11 @@ jobs:
           if [ ! -z "$PIMCORE_VERSION" ]; then
                 composer require --no-update pimcore/pimcore:"${PIMCORE_VERSION}"
           fi
+          
+      - name: "Configure Composer audit block insecure to false for lowest dependencies"
+        if: matrix.dependencies == 'lowest'
```

Comment:
looks a little odd without `${{` `}}` but there's a special rule/exception for it, maybe it would be more consistent by changing to
```suggestion
        if: ${{ matrix.dependencies }} == 'lowest'
```

https://docs.github.com/en/actions/reference/workflows-and-actions/workflow-syntax#jobsjob_idif
<img width="908" height="123" alt="image" src="https://github.com/user-attachments/assets/992711c2-0267-4308-918b-8ff84e584193" />